.github/workflows/pr-build.yml

```diff
@@ -0,0 +1,47 @@
+name: Build pull request
+
+on:
+  pull_request:
+    paths:
+      - ".github/workflows/pr-build.yml"
+      - "app/**"
+      - "gradle/**"
+      - "build.graddle.kts"
+      - "gradle.properties"
+      - "settings.gradle.kts"
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          # Make sure the release step uses its own credentials:
+          # https://github.com/cycjimmy/semantic-release-action#private-packages
+          persist-credentials: false
+          fetch-depth: 0
+
+      - name: Set up JDK
+        uses: actions/setup-java@v3
+        with:
+          java-version: '17'
+          distribution: 'temurin'
```

Comment:
Are you sure JDK needs to be set up?

Reply:
Yes... but I can double check.

Reply:
https://github.com/ReVanced/revanced-manager/actions/runs/6202388269/job/16841036450#step:4:35

I guess it needs to be setup.